debian-isogen/Makefile

```diff
@@ -0,0 +1,8 @@
+SHELL := /bin/bash
+.PHONY: test
+test: test
```

Comment:
why do we need test twice here?
I'd replace shell scripts that checking files with separate targets in Makefile